app/helpers/work_packages_helper.rb

```diff
@@ -327,22 +327,33 @@ def work_package_show_attributes(work_package)
     group_work_package_attributes work_package_show_attribute_list(work_package)
   end
 
-  def work_package_show_table_row(attribute, klass = nil, attribute_lang = nil, value_lang = nil, &block)
```

Comment:
@NobodysNightmare good catch. It honestly didn't occur to me that this is _part of our (semi-)public plugin API), so I didn't grep for usages. In the future, we really need to document the Ruby plugin API.